docs/openrpc.json

```diff
@@ -0,0 +1,917 @@
+            "type": "number"
+          },
+          "total_utxos_available": {
+            "type": "number"
+          },
+          "total_amount_locked": {
+            "type": "number"
+          },
+          "total_utxos_locked": {
+            "type": "number"
+          },
+          "utxos": {
+            "type": "array",
+            "items": {
+              "$ref": "#/components/schemas/UtxoInfo"
+            }
+          }
+        }
+      },
+      "UtxoInfo": {
+        "type": "object",
+        "properties": {
+          "address": {
+            "$ref": "#/components/schemas/Address"
+          },
+          "amount": {
+            "type": "number"
+          },
+          "tx_id": {
+            "description": "32 bytes encoded in hexadecimal.",
+            "type": "string",
+            "pattern": "pattern": "^[a-fA-F0-9]{64}$"
+          },
+          "locked": {
+            "type": "boolean"
+          },
+          "index": {
+            "type": "number"
+          }
+        }
+      },
+      "AddressInfoObject": {
+        "type": "object",
+        "properties": {
+          "address": {
+            "$ref": "#/components/schemas/Address"
+          },
+          "index": {
+            "title": "Derivation index of the address",
+            "type": "number"
+          },
+          "addressPath": {
+            "title": "Path of the address",
+            "type": "string"
+          },
+          "info": {
+            "title": "Optional extra info when getting address info",
+            "type": "null"
+          }
+        }
+      },
+      "NanoContractAction": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "enum": ["deposit", "withdrawal"]
+          },
+          "token": {
+            "$ref": "#/components/schemas/Token"
+          },
+          "amount": {
+            "$ref": "#/components/schemas/OutputValueType"
+          },
+          "address": {
+            "description": "When action type is 'withdrawal', address is required and specifies the destination for sending the output. When action type is 'deposit', address is optional and specifies the source from UTXOs.",
+            "$ref": "#/components/schemas/Address"
+          },
+          "changeAddress": {
+            "description": "When action type is 'deposit', change addess is optional and specifies the destination for the change output after selecting UTXOs. When action type is 'withdrawal', it is ignored.",
+            "$ref": "#/components/schemas/Address"
+          }
+        }
+      },
+      "TransactionHex": {
+        "description": "Transaction encoded in hexadecimal.",
+        "type": "string"
+      },
+      "NanoContract": {
+        "type": "object",
+        "allOf": [
+          {
+            "$ref": "#/components/schemas/Transaction"
+          },
+          {
+            "properties": {
+              "id": {
+                "type": "string"
+              },
```

Comment:
What's this id?

Reply:
I'm not sure, but I believe `id` corresponds to:
- blueprint id when method is initiliaze.
- nc_id otherwise.

It seems to behave as follows:
https://github.com/HathorNetwork/hathor-wallet-lib/blob/master/src/nano_contracts/nano_contract.ts

And:
https://github.com/HathorNetwork/hathor-core/blob/feat/nano-sdk-v1/hathor/nanocontracts/nanocontract.py

I added the description to make it clear.